README.md

```diff
@@ -5,6 +5,7 @@
 - Documentation: https://www.terraform.io/docs/providers/google/index.html
 - [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
 - Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
+- Issues have been turned off for this repository. To report a bug, [file an issue in terraform-provider-google repository](https://github.com/hashicorp/terraform-provider-google/issues/new/choose).
```

Comment:
While useful, this doesn't seem to fit the style of the rest of the list. What about something like:
- Issue tracker: [link to TPG issues]

Reply:
@c2thorn point taken and update made. It's in the same style as the rest of the list now